### DISCUSSION TITLE
Welcome to loxs Discussions!

### DISCUSSION THREAD

**Discussion by coffinxp** [coffinxp/loxs#48] [Announcements]
# Welcome [_**`Lostsec`**_](https://t.me/lostsec) community!

  We are using **Discussions** as a place to connect with other members of our community. We hope that you:
  * Ask questions and doubts you’re wondering about.
  * Share ideas (We will implement better ideas in [`loxs`](https://github.com/coffinxp/loxs))!
  * Respectfully engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together **<3**

  To get started, comment below with an introduction of yourself ;)

**Comment by 1hehaq** [comment]
Hello team ✌🏿

**Reply by LB1941** [reply]
Hi,

Since the templates are are for nuclei are always being updated:
I mentioned to Coffe:
What if you added ninja+lostsec+loxs+nuclei...and since now there is talk of CLI+maybe add caido code to it also?
I also mentioned turning it into a podman image so it can be run on any system where podman desktop is installed?
Just ideas.
If no one likes the idea(s)..let me know why...

**Reply by LB1941** [reply]
Also I have a working tool to add to it for the subfinder parts....but someone might want to make it faster...it takes a long time but create a huge url list.

**Reply by LB1941** [reply]
tool to add:  docker pull ghcr.io/lb1941/myurldiscoverytool:latest
use podman:
podman pull ghcr.io/lb1941/myurldiscoverytool:latest

Ill find the command i used to run it.

**Reply by LB1941** [reply]
https://github.com/users/LB1941/packages/container/package/myurldiscoverytool

**Reply by 1hehaq** [reply]
> Hi,
> 
> Since the templates are are for nuclei are always being updated:
> I mentioned to Coffe:
> What if you added ninja+lostsec+loxs+nuclei...and since now there is talk of CLI+maybe add caido code to it also?
> I also mentioned turning it into a podman image so it can be run on any system where podman desktop is installed?
> Just ideas.
> If no one likes the idea(s)..let me know why...

Hello,

Yyh, ninjaxlso and lostsec tool also combined in this loxs...
We're planning to do something unique from other currently existing tools, later we will think about making a caido plugin, and our GUI loxs will be a surprise ;)

Thank you for your suggestions <3

**Reply by 1hehaq** [reply]
> https://github.com/users/LB1941/packages/container/package/myurldiscoverytool

Thank you, 

We will take a mock test with this, if it works perfectly or provide what we want, then we can implement it

**Comment by ghost** [comment]
hello hello ✌️

**Reply by Exploit90** [reply]
hi this is exploit 90 i am self taught ethical hacker and bug hunter wassup guys and  how are you coffin bro

**Reply by ghost** [reply]
Hey, welcome !  😁

**Reply by Exploit90** [reply]
hey bro hi dude

**Reply by 1hehaq** [reply]
> hi this is exploit 90 i am self taught ethical hacker and bug hunter wassup guys and  how are you coffin bro  

May i call you exploit, lol

btw, **welcome** 😉🪐

**Reply by ghost** [reply]
hello everyone, excited to see what you guys do in the future. keep up the hard work :)

also thank you hex for always replying so quickly to any issues.

**Reply by ghost** [reply]
hello and welcome ciznaa

yeah that's normal ;)

**Reply by 1hehaq** [reply]
> hello and welcome ciznaa
> 
> yeah that's normal ;)

Not normal, hexa is not a human 👽

**Reply by Exploit90** [reply]
> > hi this is exploit 90 i am self taught ethical hacker and bug hunter wassup guys and  how are you coffin bro
> 
> May i call you exploit, lol
> 
> btw, **welcome** 😉🪐

yeah everything is cool

**Reply by Exploit90** [reply]
hey guys i do everything in cybersec and i love osint too so i wanna ask is anyone here who do telegram osint i mean yeah for gmail i can use holehe or g hunt for photo i will use exif repo and many more and for phone number there are not so much things even phoneinfoga is also not so good and all about web scraping if you are not so much on internet you cant be doxed proper whether at all this is cyebrsec so can you guys tell

**Comment by 1hehaq** [comment]
share this link to all your group like you foward all posts from **_LostSec_** so we can get our community back ❤️

[Backup of **_LostSec_** channel](https://t.me/+BGVzIMlsr-VhZWQ1)

**Comment by Exploit90** [comment]
Hey guys i wanna run this tool in windows will it work on windows  can you guys tell @coffinxp  @1hehaq ???

**Reply by 1hehaq** [reply]
> Hey guys i wanna run this tool in windows will it work on windows  can you guys tell @coffinxp  @1hehaq ???

Sure, you can try loxs on windows..

Part of updating a patching every time i run this in powershell

**Reply by Exploit90** [reply]
alright lemme try well i have done and python is already installed in my pc still its getting close lemm try one more time it have to be run bro why not yeah :b

**Reply by LB1941** [reply]
Like this?
![ghauri6](https://github.com/user-attachments/assets/620c11ba-836e-491c-a68d-4e6dec2ce8f5)

**Reply by LB1941** [reply]
Go runs great in powershell also:
![subfinder6](https://github.com/user-attachments/assets/15e21de7-27b4-4008-8e09-04f0267e4636)
